ccdaservice/support/background_services.sql

```diff
@@ -0,0 +1,32 @@
+-- phpMyAdmin SQL Dump
+-- version 4.5.1
+-- http://www.phpmyadmin.net
+--
+-- Host: 127.0.0.1
+-- Generation Time: Mar 02, 2017 at 09:27 AM
+-- Server version: 10.1.19-MariaDB
+-- PHP Version: 5.6.28
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `openemr`
+--
+
+--
+-- Dumping data for table `background_services`
+--
+
+INSERT INTO `background_services` (`name`, `title`, `active`, `running`, `next_run`, `execute_interval`, `function`, `require_once`, `sort_order`) VALUES
+('handle-ccda-service', 'C-CDA Node Service', 1, 0, '2017-03-02 06:15:58', 1, 'runCheck', '/ccdaservice/ssmanager.php', 99);
```

Comment:
Add to database.sql and upgrade sql script. 0 for active and blank for next_run. And see below comments on having it turned on in globals. Also noted that the running flag is not being set when this process is running.